core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -341,6 +341,13 @@ trait AttributeComponent {
       }
     }
 
+    def deleteAttributeColumn(workspaceNamespace: String, workspaceName: String, entityType: String, entityNamespace: String, attributeName: String) = {
+     sql"""delete ea from ENTITY_ATTRIBUTE ea
```

Comment:
nit: I don't believe you need the `delete ea from …`, it should just be `delete from …`